selftest/cgroup-legacy/main.bpf.c

```diff
@@ -5,9 +5,6 @@
 #include <bpf/bpf_core_read.h>
 #include <bpf/bpf_endian.h>
 
-#include <netinet/ip_icmp.h>
-#include <netinet/ip.h>
-
```

Comment:
Why remove these includes? I suppose we want to keep specific requirements only in the consumers. selftests are examples of consumer use.

Reply:
And it seems not related to the iter effort.